### PR TITLE
Integrate Travis, Integrate Deploy to IBM Cloud, Update README

### PR DESCRIPTION
**Pull request by jayakrishna-duvvuri**
This PR is WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,82 @@
...
 You can find the repository for the template [here](https://github.com/BlackrockDigital/startbootstrap-sb-admin "SB Admin Repo"), including its [License](https://github.com/BlackrockDigital/startbootstrap-sb-admin/blob/master/LICENSE "SB Admin License").
 
 
-### Configure credentials
+### 6. Get IBM Cloud Services Credentials and add to .env file
+
+As you create the Blumix Services, you'll need to create service credentials and get the
+username and password:
+
+Move the watson-stock-advisor/web/env.sample file to /.env and populate the service
```
Comment (creitz) [inline README.md]: Should be "to watson-stock-advisor/web/.env and populate the service"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "test-travis",
+  "description": "Simple Travis-CI check for JSHint (Code Linting)",
```
Comment (stevemar) [inline package.json]: this should describe the package we're creating

**Review by stevemar — COMMENTED** [review]
good start so far!

File: manifest.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+memory: 256MB
+declared-services:
+  wsa-discovery:
+    label: discovery
+    plan: lite
+  wsa-cloudant:
+    label: cloudantNoSQLDB
+plan: Lite
```
Comment (stevemar) [inline manifest.yml]: this should be indented

File: manifest.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+applications:
+- path: .
+  name: watsonstockadvisor
+  buildpack: sdk-for-nodejs
+  memory: 512M
```
Comment (stevemar) [inline manifest.yml]: you can try 256 here to make sure it runs under 'lite' accounts

File: manifest.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+memory: 256MB
```
Comment (markstur) [inline manifest.yml]: memory is below.  I think you want to delete this one and make the other one 256

**Comment by stevemar** [comment]
@markstur can you take a look if you have time?

**Comment by stevemar** [comment]
epic pull request, great job @creitz and @jayakrishna-duvvuri !

**Comment by creitz** [comment]
😜

**Pull request merged**